src/clients/inst_add-on.rb

```diff
@@ -1,167 +1,3 @@
-# encoding: utf-8
```

Comment:
NP: Um, I'd avoid refactoring like this in RC phase, although it is trivial there is still a risk that we overlook something. But do not revert it, if you already did it...

Reply:
Yes, I agree. However, in this case, moving the client to a proper place (to avoid the automatic `#main` call) was the only way to write some unit tests. Otherwise, mocking did not have the desired effect.

Reply:
I see, you are right...